src/Meshes/geometry.ts

```diff
@@ -275,9 +275,12 @@ export class Geometry implements IGetSetVerticesData {
             var meshes = this._meshes;
             var numOfMeshes = meshes.length;
 
+            var minimum = this._boundingInfo?.minimum ?? this._extend.minimum;

```

Comment:
Would not this be a breaking change preventing to update bounding info if they were already present e.g. update positions as we often do in pg ?

Reply:
Also as we are not bypassing the _updateExtend we will not save anything ?

Reply:
So I think adding back the `useBoundingInfoFromGeometry` property is the safest thing to do, as the new code will only be executed if it is `true` and so no breaking change possible?

I can update `_updateExtend` like this:
```typescript
private _updateExtend(data: Nullable<FloatArray> = null) {
    if (this.useBoundingInfoFromGeometry && this._boundingInfo) {
        this._extend = {
            minimum: this._boundingInfo.minimum.clone(),
            maximum: this._boundingInfo.maximum.clone()
        }
    } else {
        if (!data) {
            data = this.getVerticesData(VertexBuffer.PositionKind)!;
        }

        this._extend = extractMinAndMax(data, 0, this._totalVertices, this.boundingBias, 3);
    }
}
```